controllers/courses.js

```diff
@@ -620,99 +622,122 @@ router.get('/:courseId/', (req, res, next) => {
 	];
 
 	// ########################### start requests to new Editor #########################
+	let editorBackendIsAlive = true;
 	if (EDITOR_URL) {
```

Comment:
This sounds like a feature, why translating the existence of an url into a feature flag?
Instead I would use a feature flag and if enabled, the url (and maybe other values as well) should be defined.

Reply:
The same i do not add it. It is the same like host and server urls. This is not a feature. The new editor is a base MS in future. But the client should work without, like calendar and content MS.